ccp/modules/dashboard-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  fhir2sql:
+    depends_on:
+      - "dashboard-db"
```

Comment:
it depends on blaze as well, no?

Reply:
true

Reply:
thanks for pointing this out. Connecting to blaze from fhir2sql turns out to be a bit more tricky than I initially expected. From what I understand fhir2sql would not be able to connect to the blaze container like this anyway because blaze is defined in a different docker-compose file and is thus in a separate network. Correct? 

One way to address this would be to create a shared network using the ```networks``` directive.

Reply:
Good question. I was under the impression, that only one docker network is created, as we `-f` all the docker compose files.

Reply:
This work docker will literally use a yaml merge lib to merge all the docker files and pretend it's one docker file